apps/eec/fields/index.js

```diff
@@ -124,89 +124,187 @@ module.exports = {
         child: 'input-text'
       },
       {
         value: 'gwf',
-        toggle: 'reporter-gwf',
+        toggle: 'requestor-gwf',
         child: 'input-text'
       },
       {
         value: 'uan',
-        toggle: 'reporter-uan',
+        toggle: 'requestor-uan',
         child: 'input-text'
       },
       {
         value: 'passport',
-        toggle: 'reporter-passport',
+        toggle: 'requestor-passport',
         child: 'input-text'
       },
       {
         value: 'no-reference'
       }
     ]
   },
-  'reporter-brp': {
+  'requestor-brp': {
     mixin: 'input-text',
     className: ['govuk-input', 'govuk-!-width-two-thirds'],
     validate: ['required', BRPValidator],
     dependent: {
-      field: 'reporter-reference-type',
+      field: 'requestor-reference-type',
       value: 'brp'
     }
   },
-  'reporter-gwf': {
+  'requestor-gwf': {
     mixin: 'input-text',
     className: ['govuk-input', 'govuk-!-width-two-thirds'],
     validate: ['required', GWFValidator],
     dependent: {
-      field: 'reporter-reference-type',
+      field: 'requestor-reference-type',
       value: 'gwf'
     }
   },
-  'reporter-uan': {
+  'requestor-uan': {
     mixin: 'input-text',
     className: ['govuk-input', 'govuk-!-width-two-thirds'],
     validate: ['required', UANValidator],
     dependent: {
-      field: 'reporter-reference-type',
+      field: 'requestor-reference-type',
       value: 'uan'
     }
   },
-  'reporter-passport': {
+  'requestor-passport': {
     mixin: 'input-text',
     className: ['govuk-input', 'govuk-!-width-two-thirds'],
     validate: ['required'],
     dependent: {
-      field: 'reporter-reference-type',
+      field: 'requestor-reference-type',
       value: 'passport'
     }
+  },
+  'requestor-contact-method': {
+    mixin: 'radio-group',
+    validate: ['required'],
+    options: [
+      {
+        value: 'email',
+        toggle: 'requestor-email',
+        child: 'input-text'
+      },
+      {
+        value: 'uk-address',
+        toggle: 'address-details-fieldset',
+        child: 'partials/address-details'
+      }
+    ]
+  },
+  'requestor-email': {
+    validate: [
+      'required',
+      'email',
+      { type: 'maxlength', arguments: 254 }
+    ],
+    dependent: {
+      field: 'requestor-contact-method',
+      value: 'email'
+    }
+  },
+  'requestor-address-line-1': {
+    validate: ['required'],
+    dependent: {
+      field: 'requestor-contact-method',
+      value: 'uk-address'
+    }
+  },
+  'requestor-address-line-2': {
+    dependent: {
+      field: 'requestor-contact-method',
+      value: 'uk-address'
+    }
+  },
+  'requestor-town-or-city': {
+    validate: ['required'],
+    dependent: {
+      field: 'requestor-contact-method',
+      value: 'uk-address'
+    }
+  },
+  'requestor-county': {
+    className: ['govuk-input', 'govuk-!-width-two-thirds'],
+    dependent: {
+      field: 'requestor-contact-method',
+      value: 'uk-address'
+    }
```

Comment:
this can be removed as there is no validation for this field

Reply:
removed